### PR TITLE
Move sidekiq enterprise environment setting to the beginning of the install for Docker 

### PR DESCRIPTION
**Pull request by accbjt**
## Description of change
We need the sidekiq enterprise environment settings in our readme doc before we do a `make up`. If someone was to set up the Docker setup from scratch they would not be able to do it. `make up` will error out saying the environment is not set.

## Testing done
Local testing

## Testing planned
None

## Acceptance Criteria (Definition of Done)
- [ ] Verify that the setup will break without this environment set and that moving this to the top will solve the issue.
#### Unique to this PR

- [ ] _Replace this line with individual tasks unique to your PR_

#### Applies to all PRs

- [ ] Appropriate logging
- [ ] Swagger docs have been updated, if applicable
- [ ] Provide link to originating GitHub issue, or connected to it via ZenHub
- [ ] Does not contain any sensitive information (i.e. PII/credentials/internal URLs/etc., in logging, hardcoded, or in specs)
- [ ] Provide which alerts would indicate a problem with this functionality (if applicable)

### CODE REVIEW

**Comment by accbjt** [comment]
@hpjaj Not sure who else needs to review this. Can you add anyone else that might need to take a look at this.

File: README.md [hunk 1]
```diff
@@ -138,11 +146,6 @@ ERROR: Service 'vets-api' failed to build: The command '/bin/bash --login -c bun
 make: *** [db] Error 1
 ```
 
-Sidekiq Enterprise is used for worker rate limiting and additional reliability. Most
```
Comment (annaswims) [inline README.md]: lines 141-155 should stay together.  If you move this up, move it all.

Comment (accbjt) [reply README.md]: Okay I'll move this also.

**Comment by accbjt** [comment]
@annaswims I updated the changes you suggested. Can you take a look at this again. Thanks

**Pull request merged**